src/index.html

```diff
@@ -17,6 +29,82 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li>
+            <a
+              href="#apple"
+              class="nav-link is-active"
```

Comment:
The class name 'nav-link' should be 'nav__link' to maintain consistency with BEM naming conventions.